framework/tasks/handlers.py

```diff
@@ -48,11 +52,8 @@ def queued_task(task):
     """
     @functools.wraps(task)
     def wrapped(*args, **kwargs):
-        if settings.USE_CELERY:
```

Comment:
Shouldn't. Docstring says can only be applied to celery tasks, which all expose a .si method. Any task can be called inline like ```TASK.si(*args, **kwargs)()```